README.md

```diff
@@ -21,68 +20,70 @@ A minimalistic Flutter game engine.
 
 ---
 
-## About 1.0.0
 
-Our goal is to release v1 soon. We are periodically launching RCs (release candidates) as we evolve
-the code, and we are already quite happy with where we are (but there might still be further
-changes).
-
-Please use this version to get a preview of the new Flame version and also to give the team feedback
-about the new structure and/or features.
-
-The `main` branch is the bleeding edge of the v1 migration. The `master-v0.x` branch is the latest
-v0 release (where we are still merging some patches and crucial fixes).
-
-The current v1 release is
-<a title="Pub" href="https://pub.dev/packages/flame/versions#prerelease" ><img src="https://img.shields.io/pub/v/flame.svg?style=popout&include_prereleases" /></a>
-on pub. The latest stable version so far is
-<a title="Pub" href="https://pub.dev/packages/flame" ><img src="https://img.shields.io/pub/v/flame.svg?style=popout" /></a>.
-Feel free to choose the one that better suits your needs.
+## Documentation
 
----
+The full documentation for Flame can be found on the 
+[docs.flame-engine.org](https://docs.flame-engine.org/).
```

Comment:
I think either

```suggestion
The full documentation for Flame can be found on 
[docs.flame-engine.org](https://docs.flame-engine.org/).
```

or

```suggestion
The full documentation for Flame can be found on the
[Docs page](https://docs.flame-engine.org/).
```

Reply:
Fixed, took the first one so that people will see our site more often, and hopefully remember it in the future.

Reply:
agreed, just the `the` was weird